.gitattributes

```diff
@@ -0,0 +1,2 @@
+/.idea export-ignore
```

Comment:
We've got rid of `composer.lock` in https://github.com/bunq/sdk_php/pull/26 after all. Could you please drop it here as well? 

Also, don't we want to `export-ignore` the `CONTRIBUTING.md` and `/test/`, just like shown here: https://blog.madewithlove.be/post/gitattributes/?